apache-rat-tasks/pom.xml

```diff
@@ -80,6 +80,12 @@
       <artifactId>junit-jupiter-params</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
What is this dependency used for?

Reply:
I found the NotNull in the generated BaseAntTask - what is it good for in the generated file/what does it prevent me from doing?